src/loki/reach.cc

```diff
@@ -62,7 +68,7 @@ directed_reach Reach::operator()(const DirectedEdge* edge,
   // seed the expansion with a place to start expanding from
   Clear();
   graph_tile_ptr tile, start_tile = reader.GetGraphTile(edge_id);
-  if ((tile = start_tile) && costing->Filter(edge, tile) > 0.f && !edge->restrictions())
+  if ((tile = start_tile) && costing->Allowed(edge, tile, sif::kDisallowSimpleRestriction))
```

Comment:
tons of name changes like this all throughout the PR. i couldnt live with the difference in naming anymore